### PR TITLE
NAND-240 adding delete all resources script

### PR DESCRIPTION
**Pull request by bnorris2**
NAND-240 adding delete all resources script

### CODE REVIEW

File: query/src/test/java/ResourceLoader.java [hunk 1]
```diff
@@ -138,4 +142,46 @@ public void putBundle(String uri, Path path) throws Exception {
...
+    @Test
+    @Ignore
+    public void deleteAllResources() throws IOException, InterruptedException {
+        HttpClient client = HttpClient.newBuilder().build();
+        List<String> patientIds = getResourceIds("Patient");
+        List<String> bundleIds = getResourceIds("Bundle");
+        List<String> conditionIds = getResourceIds("Condition");
+        deleteResource("Patient", patientIds);
+        deleteResource("Bundle", bundleIds);
+        deleteResource("Condition", conditionIds);
```
Comment (seanmcilvenna) [inline query/src/test/java/ResourceLoader.java:185]: Add `Encounter`, `Coverage`, `MedicationRequest`, `Observation`, `Procedure`, `Location`

Comment (seanmcilvenna) [reply query/src/test/java/ResourceLoader.java:185]: and `AllergyIntolerance`

File: query/src/test/java/ResourceLoader.java [hunk 2]
```diff
@@ -138,4 +142,46 @@ public void putBundle(String uri, Path path) throws Exception {
...
+
+        bundle.getEntry().forEach(entry -> {
+            System.out.println(entry.getResource().getIdElement().getIdPart());
+            idList.add(entry.getResource().getIdElement().getIdPart());
+        });
+        return idList;
+    }
+
+    private void deleteResource(String resourceType, List<String> idList) throws IOException, InterruptedException {
+        HttpClient client = HttpClient.newBuilder().build();
```
Comment (seanmcilvenna) [inline query/src/test/java/ResourceLoader.java:165]: Can we (reasonably) move this to the CLI as a separate main() method that accepts the "fhirServerBase" as a parameter? Would like to be able to run this without IntelliJ, and run it against various servers without changing code...

**Review by seanmcilvenna — CHANGES_REQUESTED** [review]

File: query/src/test/java/ResourceLoader.java (hunk 1 above)
Comment (bnorris2) [reply query/src/test/java/ResourceLoader.java:215]: i've added these...and the expunge

**Review by seanmcilvenna — APPROVED** [review]

**Pull request merged**